packages/volto/src/components/manage/Form/Form.jsx

```diff
@@ -927,6 +945,8 @@ class Form extends Component {
                         ),
                         ...map(item.fields, (field, index) => (
                           <Field
+                            widgets={this.props.widgets}
+                            component={this.props.component}
```

Comment:
If I understand correctly, `this.props.component` is a Form component of some kind here, either the default or a provided one, so maybe this should not be passed to the Field component, right?

Reply:
This is a way to use a high level component (wrapper widget) with logic and a low level widget with just the widget markup (like the widgets in plone/components). You can check the wrapper components in the 4.x.x branch of volto-form-block.

Reply:
I don't get it, maybe I need to see an example. Let's see it very briefly at the Bucharest sprint.

Reply:
It's removed for now.